src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useState } from 'react';
+
+export const LoginView = ({ onLoggedIn }) => {
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleSubmit = (event) => {
+        // This prevent default behavior
+        event.preventDefault();
+
+        const data = {
+            Username: username,
+            Password: password
+        };
+
+        fetch('https://myflix-88009.herokuapp.com/login', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(data)
+        })
+            .then((response) => response.json())
+            .then((data) => {
+                console.log('Login response: ', data);
+                if (data.user) {
+                    localStorage.setItem('user', JSON.stringify(data.user));
+                    localStorage.setItem('token', data.token);
+                    onLoggedIn(data.user, data.token);
+                } else {
+                    alert('No such user');
+                }
+            })
+            .catch((e) => {
+                console.log(JSON.stringify(data))
```

Comment:
When your API encounters an error, the stack trace will be available in the `e` parameter.  Within your catch statement, variable `data` will only log request body i.e username and password. In this case, you can use `console.error("login error: ", e);`